### ISSUE TITLE
Finetuning on custom dataset !

### ISSUE THREAD

**Issue by vikramguptai** [facebookresearch/SlowFast#500]
Hi,

Thanks for sharing this amazing code :)  

We want to finetune pre-trained X3D model on our custom dataset with 34 classes. We made the following changes and wanted to check if these changes are sufficient for finetuning the complete model on a new dataset.

```
TRAIN.CHECKPOINT_EPOCH_RESET True # Make this True else scaler throws dimension mismatch error 
TRAIN.CHECKPOINT_FILE_PATH x3d_l.pyth # Pretrained checkpoint from model zoo
MODEL.NUM_CLASSES  34 # Specify number of classes in our dataset
```

Thanks Again !

**Comment by Balakishan77** [comment]
@vikramguptai I think while fine-tuning parameters better to use a smaller learning rate as these model parameters were obtained via pretraining, they are effective and when training the output layer from scratch use a larger learning rate.
